### DISCUSSION TITLE
Contribution support

### DISCUSSION THREAD

**Discussion by geromegrignon** [realworld-apps/realworld#1261] [Community]
Please ask any question related to contribution workflow, from implementation to git workflow.

**Comment by Aakash017** [comment]
Hi Team Can you please explain https://github.com/gothinkster/realworld/issues/1245 I would like to work on this

**Reply by geromegrignon** [reply]
Hi Aakash, based on Jake and mine explanations there, what do you need details on?
I'll be happy to support you for this contribution.

**Reply by Aakash017** [reply]
Hi Team , sorry but I am will not able to contribute in this issue as i am occupied with some other priority tasks.
